src/components/home/Home.js

```diff
@@ -17,17 +17,36 @@ export default function Home({ token }) {
       const link = pet.primary_photo_cropped;
       return (
         <Col md={4} xs={12} key={pet.id}>
-          <Card>
-            <Card.Img variant="top" alt={link ? pet.type : `${pet.type} placeholder`} src={link ? link.medium : placeholder} />
-            <Card.Body>
-              <Card.Title>{pet.name}</Card.Title>
+          <Card style={{ margin: 10, paddingTop: 10, width: "100%" }}>
+            <Card.Img
+              variant="top"
+              alt={link ? pet.type : `${pet.type} placeholder`}
+              src={link ? link.medium : placeholder}
+              style={{
```

Comment:
put this part in `home.css`
```
.home__container .card-img-top {
//css here

}
```

Reply:
Tried adding style here but it doesn't reflect on the application.